src/Symfony/Component/Mime/DraftEmail.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Mime;
+
+use Symfony\Component\Mime\Header\Headers;
+
+/**
+ * @author Kevin Bond <kevinbond@gmail.com>
+ */
+class DraftEmail extends Email
+{
+    public function __construct()
+    {
+        parent::__construct();
+
+        $this->getHeaders()->addTextHeader('X-Unsent', '1');
+    }
+
+    public function getPreparedHeaders(): Headers
```

Comment:
Let's add a comment to explain why we override the parent and why we need to keep some logic and get rid of the rest.